utils/utils_aws_manual.py

```diff
@@ -15,3 +20,51 @@ def list_hosted_zones_manual_scan():
         hosted_zones_list = hosted_zones_list + hosted_zones
 
     return hosted_zones_list
+
+
+# Given the bucket's direct URL or S3 website URL, returns True if the bucket does not exist
+def bucket_does_not_exist(bucket_url):
+    try:
+        # NOTE: We don't verify TLS, because the certificate matches the CloudFront's distribution domain name, not the S3 bucket's domain name
```

Comment:
```suggestion
```

Reply:
thanks - sorry if I'm slow to respond today - in the middle of delivering cloud security training!

Reply:
I've pulled the latest branch and retested and I now get this error, which I didn't see before:
`paul@laptop domain-protect % python manual_scans/aws/aws_cname_cloudfront_s3.py
Traceback (most recent call last):
  File "/Users/paul/src/github.com/domain-protect/domain-protect/manual_scans/aws/aws_cname_cloudfront_s3.py", line 5, in <module>
    import dns.resolver
ModuleNotFoundError: No module named 'dns'
paul@laptop domain-protect % python manual_scans/aws/aws_cname_cloudfront_s3.py`